### PR TITLE
[ISSUE #3654] Code reuse is extracted as a method.[EventMeshTestUtils]

### LINKED ISSUE #3654: [Enhancement] Code reuse is extracted as a method.[EventMeshTestUtils]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/eventmesh/issues?q=is%3Aissue) and found no similar issues.

### Enhancement Request

image
image
image
located at:
eventmesh-examples/src/main/java/org/apache/eventmesh/tcp/common/EventMeshTestUtils.java line 54-66,72-84,89-92,96-99,103-106,110-113,117-120,124-131,136-143,147-154,158-165

### Describe the solution you'd like

image
image
image
image
image
image

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by monish-byte**
Fixes #3654.

### Motivation

*New methods are created getUserAgent, getPackageMsg, getEventMeshMessage.*
*the code was reusable so extracted as method.*

### Modifications

*Describe the modifications you've done.*

### Documentation

- Does this pull request introduce a new feature? (yes / no)
- If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
- If a feature is not applicable for documentation, explain why?
- If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

File: eventmesh-examples/src/main/java/org/apache/eventmesh/tcp/common/EventMeshTestUtils.java [hunk 1]
```diff
@@ -38,6 +38,9 @@
 import java.util.UUID;
 import java.util.concurrent.ThreadLocalRandom;
 
+import javax.lang.model.element.PackageElement;
```
Comment (Alonexc) [inline eventmesh-examples/src/main/java/org/apache/eventmesh/tcp/common/EventMeshTestUtils.java:41]: Just extracting the duplicate code as a method, why would you use these two imports?

File: eventmesh-examples/src/main/java/org/apache/eventmesh/tcp/common/EventMeshTestUtils.java [hunk 2]
```diff
@@ -49,120 +52,101 @@ public class EventMeshTestUtils {
 
     private static final String DEFAULT_TTL_MS = "30000";
 
+
+    private static UserAgent getUserAgent(Integer port, String subsystem1, Integer pid){
```
Comment (Alonexc) [inline eventmesh-examples/src/main/java/org/apache/eventmesh/tcp/common/EventMeshTestUtils.java:56]: subsystem1->subsystem

File: eventmesh-examples/src/main/java/org/apache/eventmesh/tcp/common/EventMeshTestUtils.java [hunk 3]
```diff
@@ -49,120 +52,101 @@ public class EventMeshTestUtils {
...
+        final UserAgent agent = getUserAgent(UtilsConstants.PORT_2, UtilsConstants.SUB_SYSTEM_2, UtilsConstants.PID_2);
         return MessageUtils.generateSubClient(agent);
     }
 
-    public static Package syncRR() {
+    private static Package getPackageMsg(Command requestToServer, EventMeshMessage eventMeshMessage){
         final Package msg = new Package();
-        msg.setHeader(new Header(Command.REQUEST_TO_SERVER, 0, null, generateRandomString(SEQ_LENGTH)));
-        msg.setBody(generateSyncRRMqMsg());
+        msg.setHeader(new Header(requestToServer,0,null,generateRandomString(SEQ_LENGTH)));
```
Comment (Alonexc) [inline eventmesh-examples/src/main/java/org/apache/eventmesh/tcp/common/EventMeshTestUtils.java:87]: Leave a space after the comma.

File: eventmesh-examples/src/main/java/org/apache/eventmesh/tcp/common/EventMeshTestUtils.java (hunk 1 above)
Comment (monish-byte) [reply eventmesh-examples/src/main/java/org/apache/eventmesh/tcp/common/EventMeshTestUtils.java]: well I think I am using vscode. And it has auto import feature. I think thats the reason.

File: eventmesh-examples/src/main/java/org/apache/eventmesh/tcp/common/EventMeshTestUtils.java [hunk 4]
```diff
@@ -49,120 +51,101 @@ public class EventMeshTestUtils {
...
-        final EventMeshMessage mqMsg = new EventMeshMessage();
-        mqMsg.setTopic(ExampleConstants.EVENTMESH_TCP_SYNC_TEST_TOPIC);
-        mqMsg.getProperties().put(UtilsConstants.MSG_TYPE, "persistent");
-        mqMsg.getProperties().put(UtilsConstants.TTL, DEFAULT_TTL_MS);
-        mqMsg.getProperties().put(UtilsConstants.KEYS, generateRandomString(16));
-        mqMsg.getHeaders().put(Constants.DATA_CONTENT_TYPE, "text/plain");
-        mqMsg.setBody("testSyncRR");
-        return mqMsg;
+        
+        return EventMeshMessage(ExampleConstants.EVENTMESH_TCP_SYNC_TEST_TOPIC , UtilsConstants.MSG_TYPE , "persistent",UtilsConstants.KEYS, generateRandomString(16),
```
Comment (Alonexc) [inline eventmesh-examples/src/main/java/org/apache/eventmesh/tcp/common/EventMeshTestUtils.java:128]: Here it should be the `getEventMeshMessage()` method.

File: eventmesh-examples/src/main/java/org/apache/eventmesh/tcp/common/EventMeshTestUtils.java [hunk 5]
```diff
@@ -49,120 +51,101 @@ public class EventMeshTestUtils {
...
     private static EventMeshMessage generateAsyncRRMqMsg() {
-        final EventMeshMessage mqMsg = new EventMeshMessage();
-        mqMsg.setTopic(ExampleConstants.EVENTMESH_TCP_SYNC_TEST_TOPIC);
-        mqMsg.getProperties().put(UtilsConstants.REPLY_TO, "localhost@ProducerGroup-producerPool-9-access#V1_4_0#CI");
-        mqMsg.getProperties().put(UtilsConstants.TTL, DEFAULT_TTL_MS);
-        mqMsg.getProperties().put(UtilsConstants.PROPERTY_MESSAGE_REPLY_TO, "notnull");
-        mqMsg.getHeaders().put(Constants.DATA_CONTENT_TYPE, "text/plain");
-        mqMsg.setBody("testAsyncRR");
-        return mqMsg;
+        return EventMeshMessage(ExampleConstants.EVENTMESH_TCP_SYNC_TEST_TOPIC, UtilsConstants.REPLY_TO , "localhost@ProducerGroup-producerPool-9-access#V1_4_0#CI" ,
```
Comment (Alonexc) [inline eventmesh-examples/src/main/java/org/apache/eventmesh/tcp/common/EventMeshTestUtils.java:134]: Here it should be the `getEventMeshMessage()` method.

File: eventmesh-examples/src/main/java/org/apache/eventmesh/tcp/common/EventMeshTestUtils.java [hunk 6]
```diff
@@ -49,120 +51,101 @@ public class EventMeshTestUtils {
...
     public static EventMeshMessage generateBroadcastMqMsg() {
-        final EventMeshMessage mqMsg = new EventMeshMessage();
-        mqMsg.setTopic(ExampleConstants.EVENTMESH_TCP_ASYNC_TEST_TOPIC);
-        mqMsg.getProperties().put(UtilsConstants.REPLY_TO, "localhost@ProducerGroup-producerPool-9-access#V1_4_0#CI");
-        mqMsg.getProperties().put(UtilsConstants.TTL, DEFAULT_TTL_MS);
-        mqMsg.getProperties().put(UtilsConstants.PROPERTY_MESSAGE_REPLY_TO, "notnull");
-        mqMsg.getHeaders().put(Constants.DATA_CONTENT_TYPE, "text/plain");
-        mqMsg.setBody(ASYNC_MSG_BODY);
-        return mqMsg;
+        return EventMeshMessage(ExampleConstants.EVENTMESH_TCP_ASYNC_TEST_TOPIC, UtilsConstants.REPLY_TO , "localhost@ProducerGroup-producerPool-9-access#V1_4_0#CI" ,
```
Comment (Alonexc) [inline eventmesh-examples/src/main/java/org/apache/eventmesh/tcp/common/EventMeshTestUtils.java:146]: Here it should be the `getEventMeshMessage()` method.

**Comment by Alonexc** [comment]
image
please check checkstyle.

**Comment by Alonexc** [comment]
@monish-byte 
Download this plugin：
image

Import the checkstyle of the local project and check the box.
image
image
image

Following this step will solve the problem.

**Comment by monish-byte** [comment]
hey @Alonexc I have applied the changes you asked for. What should be the next step?

**Comment by Alonexc** [comment]
image
Select eventmesh checkstyle, then click that button on the left, which will usually check automatically.

**Review by xwm1992 — APPROVED** [review]

**Pull request merged**